Package.resolved

```diff
@@ -5,17 +5,17 @@
         "package": "swift-argument-parser",
         "repositoryURL": "https://github.com/apple/swift-argument-parser.git",
         "state": {
-          "branch": null,
-          "revision": "8d31a0905c346a45c87773ad50862b5b3df8dff6",
-          "version": "0.0.4"
+          "branch": "master",
+          "revision": "eb51f949cdd0c9d88abba9ce79d37eb7ea1231d0",
+          "version": null
```

Comment:
Now that your dependencies are on branches rather than tags/versions, does it still make sense to have Package.resolved file checked in? Our CI system will build this against ToT for each dependency, but if a user checks it out or uses it as a dependency they will get whatever version is stamped here.  That seems like it will cause mismatches.

Reply:
@allevato Is it safe to delete this?

Reply:
Yeah, it should be deleted *and* added to `.gitignore`. Swift Package Manager will regenerate it when it does the checkout, but if it exists already it will be used, so we don't want wrong revisions to leak to users.

Reply:
Please make sure to add Package.resolved to `.gitignore` as well.

Reply:
Overlooked that, thanks.